Source/Extensions/Blazorise.DataGrid/Enums/DataGridCellFilterType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Blazorise.DataGrid;
+
+/// <summary>
+/// Defines the DataGrid cell filter type.
+/// </summary>
+public enum DataGridCellFilterType
+{
+    Single,
+    Range1,
```

Comment:
Range 1 and 2 are quite generic. Maybe `RangeFrom` and `RangeTo` might be better?

Reply:
Seems like a good idea. Only thing is that if we potentially had multiple values in the future, like 1,2,3,4... but I don't think so. So I'll go with your suggestion.